lib/feature/add_seed_to_app/create_seed/view/create_seed_view.dart

```diff
@@ -0,0 +1,208 @@
+import 'package:app/feature/add_seed_to_app/create_seed/create_seed.dart';
+import 'package:app/generated/assets.gen.dart';
+import 'package:app/l10n/l10n.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:ui_components_lib/ui_components_lib.dart';
+
+/// Callback that calls if user taps skip checking or check phrase.
+typedef CreateSeedNavigationCallback = void Function(
+  BuildContext context,
```

Comment:
I think, we cant remove it thanks to *Page widget, but in general I don't think it's a problem because it's like a navigation callback

Reply:
Any **navigation** callback should be done centralized, through go_router and NavigationService (Otherwise, we can get collisions).

Specially this one and other *NavigationCallback:

- CheckSeedPhraseNavigationCallback
- CreatePasswordWidgetNavigationCallback
- CreateSeedNavigationCallback
- EnterSeedNameNavigationCallback
- EnterPhraseNavigationCallback

seems like not navigation but user action callback, for example, onCreateSeed(). Yeah, its just a naming convention, but we want to be consistent, isn't it? :)

Concerning context propagation: I think its a really bad idea to put context in the callback: theoretically, we have no reason to believe that widget's lifecycle is not over when we execute the callback. As I see, currently context uses only for navigation. For example:

```dart
skipCallback: (BuildContext context, List<String> phrase) {
          context.goFurther(
            AppRoute.createSeedPassword.path,
            extra: CreateSeedRouteExtra(phrase: phrase),
          );
        },
```

and can (should?) be replaced by current widget's context:

```dart
  Widget build(BuildContext context) {
    return Scaffold(
      resizeToAvoidBottomInset: false,
      appBar: const OnboardingAppBar(),
      body: CreateSeedView(
        checkCallback: (BuildContext context, List<String> phrase) {
          context.goFurther(
            AppRoute.checkSeed.path,
            extra: CreateSeedRouteExtra(phrase: phrase),
          );
        },
        skipCallback: (List<String> phrase) {
          context.goFurther(
            AppRoute.createSeedPassword.path,
            extra: CreateSeedRouteExtra(phrase: phrase),
          );
        },
      ),
    );
  }
```

Reply:
-